src/main/java/games/strategy/engine/framework/map/download/DownloadMapsWindow.java

```diff
@@ -294,35 +298,33 @@ private static void updateMapUrlAndSizeLabel(final DownloadFileDescription map,
   }
 
   private static String createLabelText(final MapAction action, final DownloadFileDescription map) {
-    final String DOUBLE_SPACE = "&nbsp;&nbsp;";
+    final String doubleSpace = "&nbsp;&nbsp;";
 
-    final long mapSize;
-    String labelText = "<html>" + map.getMapName() + DOUBLE_SPACE + " v" + map.getVersion();
+    final StringBuilder sb = new StringBuilder();
```

Comment:
It seemed cleaner to use a `StringBuilder` here instead of numerous string concatenations.